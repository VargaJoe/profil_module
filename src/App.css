/*! normalize.css v8.0.0 | MIT License | github.com/necolas/normalize.css */
/* Document
   ========================================================================== */
/**
 * 1. Correct the line height in all browsers.
 * 2. Prevent adjustments of font size after orientation changes in iOS.
 */
html {
  line-height: 1.15;
  /* 1 */
  -webkit-text-size-adjust: 100%;
  /* 2 */
}

/* Sections
   ========================================================================== */
/**
 * Remove the margin in all browsers.
 */
body {
  margin: 0;
}

/**
 * Correct the font size and margin on `h1` elements within `section` and
 * `article` contexts in Chrome, Firefox, and Safari.
 */
h1 {
  font-size: 2em;
  margin: 0.67em 0;
}

/* Grouping content
   ========================================================================== */
/**
 * 1. Add the correct box sizing in Firefox.
 * 2. Show the overflow in Edge and IE.
 */
hr {
  -webkit-box-sizing: content-box;
          box-sizing: content-box;
  /* 1 */
  height: 0;
  /* 1 */
  overflow: visible;
  /* 2 */
}

/**
 * 1. Correct the inheritance and scaling of font size in all browsers.
 * 2. Correct the odd `em` font sizing in all browsers.
 */
pre {
  font-family: monospace, monospace;
  /* 1 */
  font-size: 1em;
  /* 2 */
}

/* Text-level semantics
   ========================================================================== */
/**
 * Remove the gray background on active links in IE 10.
 */
a {
  background-color: transparent;
}

/**
 * 1. Remove the bottom border in Chrome 57-
 * 2. Add the correct text decoration in Chrome, Edge, IE, Opera, and Safari.
 */
abbr[title] {
  border-bottom: none;
  /* 1 */
  text-decoration: underline;
  /* 2 */
  -webkit-text-decoration: underline dotted;
          text-decoration: underline dotted;
  /* 2 */
}

/**
 * Add the correct font weight in Chrome, Edge, and Safari.
 */
b,
strong {
  font-weight: bolder;
}

/**
 * 1. Correct the inheritance and scaling of font size in all browsers.
 * 2. Correct the odd `em` font sizing in all browsers.
 */
code,
kbd,
samp {
  font-family: monospace, monospace;
  /* 1 */
  font-size: 1em;
  /* 2 */
}

/**
 * Add the correct font size in all browsers.
 */
small {
  font-size: 80%;
}

/**
 * Prevent `sub` and `sup` elements from affecting the line height in
 * all browsers.
 */
sub,
sup {
  font-size: 75%;
  line-height: 0;
  position: relative;
  vertical-align: baseline;
}

sub {
  bottom: -0.25em;
}

sup {
  top: -0.5em;
}

/* Embedded content
   ========================================================================== */
/**
 * Remove the border on images inside links in IE 10.
 */
img {
  border-style: none;
}

/* Forms
   ========================================================================== */
/**
 * 1. Change the font styles in all browsers.
 * 2. Remove the margin in Firefox and Safari.
 */
button,
input,
optgroup,
select,
textarea {
  font-family: inherit;
  /* 1 */
  font-size: 100%;
  /* 1 */
  line-height: 1.15;
  /* 1 */
  margin: 0;
  /* 2 */
}

/**
 * Show the overflow in IE.
 * 1. Show the overflow in Edge.
 */
button,
input {
  /* 1 */
  overflow: visible;
}

/**
 * Remove the inheritance of text transform in Edge, Firefox, and IE.
 * 1. Remove the inheritance of text transform in Firefox.
 */
button,
select {
  /* 1 */
  text-transform: none;
}

/**
 * Correct the inability to style clickable types in iOS and Safari.
 */
button,
[type="button"],
[type="reset"],
[type="submit"] {
  -webkit-appearance: button;
}

/**
 * Remove the inner border and padding in Firefox.
 */
button::-moz-focus-inner,
[type="button"]::-moz-focus-inner,
[type="reset"]::-moz-focus-inner,
[type="submit"]::-moz-focus-inner {
  border-style: none;
  padding: 0;
}

/**
 * Restore the focus styles unset by the previous rule.
 */
button:-moz-focusring,
[type="button"]:-moz-focusring,
[type="reset"]:-moz-focusring,
[type="submit"]:-moz-focusring {
  outline: 1px dotted ButtonText;
}

/**
 * Correct the padding in Firefox.
 */
fieldset {
  padding: 0.35em 0.75em 0.625em;
}

/**
 * 1. Correct the text wrapping in Edge and IE.
 * 2. Correct the color inheritance from `fieldset` elements in IE.
 * 3. Remove the padding so developers are not caught out when they zero out
 *    `fieldset` elements in all browsers.
 */
legend {
  -webkit-box-sizing: border-box;
          box-sizing: border-box;
  /* 1 */
  color: inherit;
  /* 2 */
  display: table;
  /* 1 */
  max-width: 100%;
  /* 1 */
  padding: 0;
  /* 3 */
  white-space: normal;
  /* 1 */
}

/**
 * Add the correct vertical alignment in Chrome, Firefox, and Opera.
 */
progress {
  vertical-align: baseline;
}

/**
 * Remove the default vertical scrollbar in IE 10+.
 */
textarea {
  overflow: auto;
}

/**
 * 1. Add the correct box sizing in IE 10.
 * 2. Remove the padding in IE 10.
 */
[type="checkbox"],
[type="radio"] {
  -webkit-box-sizing: border-box;
          box-sizing: border-box;
  /* 1 */
  padding: 0;
  /* 2 */
}

/**
 * Correct the cursor style of increment and decrement buttons in Chrome.
 */
[type="number"]::-webkit-inner-spin-button,
[type="number"]::-webkit-outer-spin-button {
  height: auto;
}

/**
 * 1. Correct the odd appearance in Chrome and Safari.
 * 2. Correct the outline style in Safari.
 */
[type="search"] {
  -webkit-appearance: textfield;
  /* 1 */
  outline-offset: -2px;
  /* 2 */
}

/**
 * Remove the inner padding in Chrome and Safari on macOS.
 */
[type="search"]::-webkit-search-decoration {
  -webkit-appearance: none;
}

/**
 * 1. Correct the inability to style clickable types in iOS and Safari.
 * 2. Change font properties to `inherit` in Safari.
 */
::-webkit-file-upload-button {
  -webkit-appearance: button;
  /* 1 */
  font: inherit;
  /* 2 */
}

/* Interactive
   ========================================================================== */
/*
 * Add the correct display in Edge, IE 10+, and Firefox.
 */
details {
  display: block;
}

/*
 * Add the correct display in all browsers.
 */
summary {
  display: list-item;
}

/* Misc
   ========================================================================== */
/**
 * Add the correct display in IE 10+.
 */
template {
  display: none;
}

/**
 * Add the correct display in IE 10.
 */
[hidden] {
  display: none;
}

/*
Flaticon icon font: Flaticon
Creation date: 09/02/2018 13:02
*/
@font-face {
  font-family: "Flaticon";
  src: local("Flaticon"), url("./fonts/Flaticon.eot");
  src: local("Flaticon"), url("./fonts/Flaticon.eot?#iefix") format("embedded-opentype"), url("./fonts/Flaticon.woff") format("woff"), url("./fonts/Flaticon.ttf") format("truetype"), url("./fonts/Flaticon.svg#Flaticon") format("svg");
  font-weight: normal;
  font-style: normal;
}
@media screen and (-webkit-min-device-pixel-ratio: 0) {
  @font-face {
    font-family: "Flaticon";
    src: local("Flaticon"), url("./fonts/Flaticon.svg#Flaticon") format("svg");
  }
}
.fi:before {
  display: inline-block;
  font-family: "Flaticon";
  font-style: normal;
  font-weight: normal;
  font-variant: normal;
  line-height: 1;
  text-decoration: inherit;
  text-rendering: optimizeLegibility;
  text-transform: none;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
  font-smoothing: antialiased;
}

.flaticon-organizational-chart:before {
  content: "\f100";
}

.flaticon-family-tree:before {
  content: "\f101";
}

.flaticon-magnifier:before {
  content: "\f102";
}

.flaticon-double-angle-pointing-to-right:before {
  content: "\f103";
}

.flaticon-book:before {
  content: "\f104";
}

.flaticon-folded-newspaper:before {
  content: "\f105";
}

.flaticon-people:before {
  content: "\f106";
}

.flaticon-discussion:before {
  content: "\f107";
}

.flaticon-calendar:before {
  content: "\f108";
}

.flaticon-partners:before {
  content: "\f109";
}

.flaticon-lightbulb-idea:before {
  content: "\f10a";
}

.flaticon-user:before {
  content: "\f10b";
}

/* hide visual*/
.content_to_right {
  padding: 5.625rem 0 0 21.875rem;
}

.wrapp {
  max-width: 75rem;
  width: 100%;
  margin: 0 auto;
  padding: 0 0.9375rem;
  -webkit-box-sizing: border-box;
          box-sizing: border-box;
}

body {
  background-color: #edecec;
  font-family: arial;
}

.title {
  font-size: 2.25rem;
  font-weight: 500;
  padding-bottom: 0.625rem;
  border-bottom: 1px solid #000;
  width: 100%;
}
.title--description {
  margin-bottom: 0.625rem;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
      -ms-flex-pack: justify;
          justify-content: space-between;
}

a {
  color: #0aa2b5;
  text-decoration: none;
}

.btn {
  background-color: #5cb85c;
  color: #fff;
  border: 1px solid #5cb85c;
  font-size: 1rem;
  padding: 0 1.25rem;
  -webkit-box-sizing: border-box;
          box-sizing: border-box;
  cursor: pointer;
  border-radius: 0.625rem;
  text-transform: capitalize;
  line-height: 2.1875rem;
  display: inline-block;
}
.btn:hover {
  background-color: #7cc576;
}

input,
textarea {
  width: 100%;
  line-height: 1.5625rem;
  padding: 0 0.625rem;
  -webkit-box-sizing: border-box;
          box-sizing: border-box;
  margin-bottom: 0.625rem;
}

textarea {
  resize: none;
}

fieldset {
  margin-bottom: 0.625rem;
}

legend {
  padding: 0 0.625rem;
}

label {
  font-size: 0.8125rem;
  padding: 0 0.625rem;
}

.login_page {
  max-width: 22.5rem;
  margin: 0 auto;
  border: 1px solid #868686;
  padding: 0 0.9375rem 0.9375rem;
  border-radius: 0.625rem;
}

.fix_block {
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
}

.header {
  background: #fff;
  position: fixed;
  top: 0;
  left: 21.875rem;
  right: 0;
  height: 5.625rem;
  border-bottom: 1px solid #000;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
}
.header__search {
  position: relative;
  display: inline-block;
}
.header__search button {
  position: absolute;
  top: 0;
  right: 0;
  bottom: 0;
  width: 3.4375rem;
  background: none;
  border: none;
  color: #7e7b6e;
  font-size: 1.4375rem;
}
.header__search input {
  max-width: 37.5rem;
  width: 100%;
  border: 3px solid #7e7b6e;
  height: 2.6875rem;
  padding: 0 3.75rem 0 0.75rem;
  -webkit-box-sizing: border-box;
          box-sizing: border-box;
  border-radius: 0.9375rem;
  color: #7e7b6e;
}

.sidebar {
  position: fixed;
  top: 0;
  left: 0;
  bottom: 0;
  max-width: 21.875rem;
  width: 100%;
  background-color: #22262c;
  color: #fff;
  border-right: 1px solid #22262c;
  overflow-y: auto;
  padding-bottom: 1.25rem;
}
.sidebar__logo {
  height: 5.625rem;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
  -webkit-box-pack: center;
      -ms-flex-pack: center;
          justify-content: center;
  background-color: #080e1a;
}
.sidebar__user {
  background-color: #0aa2b5;
  color: #fff;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: start;
      -ms-flex-pack: start;
          justify-content: flex-start;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
  padding: 0.75rem;
  margin-bottom: 0.9375rem;
  border-top: 1px solid #60c2cf;
  border-right: 1px solid #60c2cf;
  border-bottom: 1px solid #60c2cf;
}
.sidebar__user__avatar {
  width: 6.25rem;
  height: 6.25rem;
  margin-right: 1.25rem;
  border-radius: 100%;
  overflow: hidden;
}
.sidebar__menu {
  padding: 0.9375rem 0 0;
}
.sidebar__menu__item {
  height: 3.3125rem;
  padding: 0 0.75rem;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
  -webkit-box-pack: start;
      -ms-flex-pack: start;
          justify-content: flex-start;
  -webkit-transition: all .2s ease;
  transition: all .2s ease;
}
.sidebar__menu__item__icon {
  display: inline-block;
  background-color: #000;
  width: 3rem;
  height: 3rem;
  border-radius: 100%;
  text-align: center;
  line-height: 3rem;
  vertical-align: middle;
  margin-right: 0.625rem;
  color: #868686;
  -webkit-transition: all .2s ease;
  transition: all .2s ease;
  font-size: 1.5rem;
}
.sidebar__menu__item:hover, .sidebar__menu__item.active {
  background-color: #121419;
  cursor: pointer;
  -webkit-transition: all .2s ease;
  transition: all .2s ease;
}
.sidebar__menu__item:hover .sidebar__menu__item__icon, .sidebar__menu__item.active .sidebar__menu__item__icon {
  background-color: #30353f;
  color: #fff;
}
.sidebar__menu__item:hover .sidebar__menu__item__name, .sidebar__menu__item.active .sidebar__menu__item__name {
  color: #0aa2b5;
}

.user {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
      -ms-flex-pack: justify;
          justify-content: space-between;
  -ms-flex-wrap: nowrap;
      flex-wrap: nowrap;
  -webkit-box-align: start;
      -ms-flex-align: start;
          align-items: flex-start;
  margin-bottom: 1.25rem;
}
.user__global_info {
  width: 100%;
}
.user__global_info__position {
  margin-bottom: 1.25rem;
  font-size: 1.5rem;
  font-weight: 400;
  color: #607d8b;
}
.user__global_info__list {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
      -ms-flex-pack: justify;
          justify-content: space-between;
}
.user__global_info__list > div {
  width: 50%;
}
.user__global_info__list__item {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
      -ms-flex-pack: justify;
          justify-content: space-between;
  text-align: left;
  padding: 0.3125rem 0;
}
.user__global_info__list__item > div {
  width: 50%;
}
.user__edit_global_info {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -ms-flex-wrap: wrap;
      flex-wrap: wrap;
  -webkit-box-pack: justify;
      -ms-flex-pack: justify;
          justify-content: space-between;
}
.user__edit_global_info__item {
  width: 49%;
  margin-bottom: 0.625rem;
}
.user__avatar {
  margin-right: 1.875rem;
}
.user__avatar img {
  width: 18.75rem;
  height: 18.75rem;
  border-radius: 100%;
  overflow: hidden;
}

.skils {
  list-style: none;
  margin: 0;
  padding: 0;
}
.skils li {
  display: inline-block;
  margin-right: 0.625rem;
  margin-bottom: 0.625rem;
}
.skils a {
  display: block;
  padding: 0.3125rem 0.9375rem;
  border: 3px solid #eea032;
  color: #eea032;
  border-radius: 1.25rem;
}
.skils a:hover {
  background-color: #eea032;
  color: #fff;
  border: 3px solid #ffc107;
}

.main {
  padding: 1.25rem 0;
}
