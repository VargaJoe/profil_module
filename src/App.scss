@import "scss/variables.scss";

@import "scss/normalize.scss";
@import "scss/fonts.scss";

@import "scss/mixins.scss";
@import "scss/global.scss";
@import "scss/loader.scss";

@import "scss/login.scss";
@import "scss/changePass.scss";

@import "scss/header.scss";
@import "scss/sidebar.scss";
@import "scss/profil.scss";
@import "scss/main.scss";
@import "scss/footer.scss";
@import "scss/achievement.scss";
@import "scss/skills.scss";

// responsive
@import "scss/tablet";
@import "scss/mobile";
 