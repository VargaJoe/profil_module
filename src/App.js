import React, { Component }     from 'react';
import { connect }              from 'react-redux';
import {
  Route,
  Redirect,
  withRouter
}                               from 'react-router-dom';
import { Authentication }       from 'sn-client-js';
import { Actions, Reducers }    from 'sn-redux';

import './App.css';
import './App.scss';

// import page components
import Sidebar                  from './components/Sidebar';
import Header                   from './components/Header';
import Profil                   from './components/Profil';
import EditProfil               from './components/EditProfil';
import { Login }                from './components/Login';

// import pageActions              from './actions/index';



// new action


class App extends Component {
    constructor(props) {
        super(props);
        this.state = {
            user : {}
        }

        this.formSubmit = this.formSubmit.bind(this);
    }

    formSubmit(e, email, password) {
        this.props.login(email, password);
    }

  render() {

    let myTask = this.props.repo.Load('/Root/IMS/BuiltIn/Portal/'+this.props.userName, {
        select: ['Name', 'DisplayName', "JobTitle", "Email", "FullName", "Description", "Languages", "Phone", "Gender", "BirthDate", "Education"],
    }).subscribe(loadedTask => {

        if(Object.getOwnPropertyNames(this.state.user).length === 0) {
            this.setState({ user: loadedTask});
            this.props.onAddTrack(loadedTask);
        }
    }, error => console.error);

    return (
        <div className="content_to_right">
            <Route
                path="/"
                render={routerProps => {
                    const status = this.props.loginState !== Authentication.LoginState.Authenticated;
                    return status ?
                      <Redirect key="login" to="/login" />
                      : (
                        <div>
                            <Header/>
                            <Sidebar/>
                        </div>
                      );
                }}
            />
            <Route
                exact={true}
                path="/login"
                render={routerProps => {
                    const status = this.props.loginState !== Authentication.LoginState.Authenticated;
                    return status ?
                      <Login formSubmit={this.formSubmit} />
                      : <Redirect key="dashboard" to="/" />;
                }}
            />
            <main className="main">
                <div className="wrapp">
                    <Route path={"/user/"+this.props.userName} render={ () => {
                        return <Profil  />
                    }}  />
                    <Route path="/edituser" render={ () => {
                        return <EditProfil />
                    }}  />
                </div>
            </main>
        </div>
    );
  }
} 

const mapStateToProps = (state, match) => {
  return {
    loginState: Reducers.getAuthenticationStatus(state.sensenet),
    userName : state.sensenet.session.user.userName,
  };
};

export default withRouter(connect(
    mapStateToProps,
    dispatch => ({
        onAddTrack: (usertt) => {
            dispatch({ type: 'UPDATE_LOGINED_USER', payload: usertt })
        },
        login: Actions.UserLogin,
    })
)(App));
