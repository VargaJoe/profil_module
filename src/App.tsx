import * as React                           from 'react';
import { connect }                          from 'react-redux';
import {
  Route,
  Redirect,
  Switch,
  withRouter,
}                                           from 'react-router-dom';
import { Actions }                from '@sensenet/redux';
import { LoginState }                       from '@sensenet/client-core';
// import './App.css';
import './App.scss';
import Body                                 from './components/Body';
import { Login }                            from './components/Login';

export interface AppProps {
    login: Function;
    userName: string;
    loginState: LoginState;
    store: any;
    repository: any;
}

class App extends React.Component<AppProps, any> {
    constructor(props: AppProps) {
        super(props);
        this.formSubmitHandler = this.formSubmitHandler.bind(this);
    }

    public formSubmitHandler = (email: string, password: string) => {
        this.props.login(email, password);
    }

    public render() {  
    
    // !important 
    // status - it's a boolean value about user authentication state
    const status = this.props.loginState !== LoginState.Authenticated;
    return (
        <div>
            <Switch>
                <Route 
                    exact={true}  
                    path="/login"   
                    render={routerProps => { 
                        return status ?
                                <Login formSubmit={this.formSubmitHandler} />
                            :   <Redirect key="dashboard" to="/" />;
                    }} 
                />
                <Route 
                    path="/" 
                    key="dashboard"
                    render={(routerProps) => {
                        return (<Body {...routerProps} />);
                    }} 
                />   
            </Switch> 
        </div>
    );
  } 
}

export const mapStateToProps = (state: any, match: any) => {
    return {
        loginState:     state.sensenet.session.loginState,
        userName :      state.sensenet.session.user.userName,
    };
};

export default withRouter(connect(
    mapStateToProps,
    (dispatch) => ({
        login:          (username: string, password: string) => dispatch(Actions.userLogin(username, password)),
    })
)(App as any));