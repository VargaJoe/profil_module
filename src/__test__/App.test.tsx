import * as React 						from 'react';
import App 								from '../App';
import { 
	configure, 
	shallow } 							from 'enzyme';
import * as Adapter 					from 'enzyme-adapter-react-16';
import toJson 				      		from 'enzyme-to-json';

configure( {adapter: new Adapter()} );

describe('<App /> shallow rendering', () => {
	const wrapper = shallow(<App />);
	it('Contain one H1 element ', () => {
		expect(wrapper.find('h1').length).toBe(0);
		
	});
	it('Match to snapshot', () => {
		const tree = shallow(<App />);
		expect(toJson(tree)).toMatchSnapshot();
	});

	// it('On button click change h1 text', () => {
	// 	const button = wrapper.find('button');
	// 	expect(wrapper.find('button').length).toBe(1);
	// 	expect(wrapper.find('h1').text()).toBe('No');
	// 	button.simulate('click');
	// 	expect(wrapper.find('h1').text()).toBe('Yes');
	// });

	// it('On input element change text', () => {
	// 	const input = wrapper.find('input');
	// 	expect(wrapper.find('input').length).toBe(1);
	// 	expect(wrapper.find('h2').text()).toBe('');
	// 	input.simulate('change', {target: { value: 'Loldon'}});
	// 	expect(wrapper.find('h2').text()).toBe('Loldon');
	// });

	// it('Update class name with setState', () => {
	// 	expect(wrapper.find('.blue').length).toBe(1);
	// 	expect(wrapper.find('.red').length).toBe(0);
	// 	wrapper.setState({ mainColor: 'red' });
	// 	expect(wrapper.find('.blue').length).toBe(0);
	// 	expect(wrapper.find('.red').length).toBe(1);
	// });

	// it('calls ComponentDidMouint', () => {
	// 	jest.spyOn(App.prototype, 'componentDidMount');
	// 	expect(App.prototype.componentDidMount.call.length).toBe(1);
	// });
});
