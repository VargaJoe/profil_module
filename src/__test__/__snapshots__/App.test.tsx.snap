// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> shallow rendering Match to snapshot 1`] = `
<Route
  render={[Function]}
/>
`;
